msu/utils/array.nut

```diff
@@ -104,4 +104,15 @@
 			}
 		}
 	}
+
+	// Returns a new array
+	function removeDuplicates( _array )
```

Comment:
Actually this should probably be called `uniqueValues` or `asSet` or something, as it  does not change the array in place.

Reply:
Yeah I see your point. `removeDuplicates` does sound like a function that should do its job in place. I don't like `asSet`. How about `filterUniques`?

Reply:
`filterUniques` sounds like it'll only give you duplicate values, lol.

Reply:
How about `uniques` or `getUniques`? I think `uniques` might be ok?

Reply:
I think `.uniques()` is fine actually, same idea as `.keys()` and `.values()` which I like